packages/uops-dashboard/src/server/clients/code/CodeClient.ts

```diff
@@ -0,0 +1,3 @@
+export interface CodeClient {
```

Comment:
is there a need for this interface? for now we have only type of this client

Reply:
I was planning to introduce 2nd client but there was no need for it in the end. Will remove the interface for now